### PR TITLE
added homepage with like function

### PR DESCRIPTION
**Pull request by MesakDuduCoder**
1.  I added movie cards to the body.
2.  I added like functionality.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,57 @@
...
+const likes = await updateLikes(appId);
+//  console.log(likes);
+//  console.log(typeof(likes));
+await display(movies, likes, appId);
+
+const likeBtns = document.querySelectorAll('.like-button');
+// console.log(likeBtns);
+async function update(likes, index, span) {
+  console.log(likes);
+  const movieId = `${index + 1}`;
```
Comment (yashodhicy) [inline src/index.js:38]: It is better to remove unexpected console.log statements for better practice and for preventing leaking the data.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,57 @@
...
+  if (likes.length === 0) {
+    likes = [];
+  } else {
+    likes = JSON.parse(likes);
+  }
+  return likes;
+};
+
+const likes = await updateLikes(appId);
+//  console.log(likes);
```
Comment (yashodhicy) [inline src/index.js:30]: it is better to remove commented codes for the best practice and for a clean code. ☺️

File: src/index.js [hunk 3]
```diff
@@ -1 +1,57 @@
...
+    console.log('likes');
+    if (movieId === item.item_id) {
+      span.innerHTML = `${item.likes}`;
+    }
+  });
+}
+
+likeBtns.forEach((item, index) => {
+  item.addEventListener('click', async (e) => {
+    const asd = e.currentTarget.parentNode.children[0];
```
Comment (yashodhicy) [inline src/index.js:51]: You have used async functions which is very good 👏. It is also better to add good variable names as a best practice, making it easier to understand the code later.🙌

**Review by yashodhicy — CHANGES_REQUESTED** [review]
You have done a good job in this code. 
changes

1. have used async functions
2. have use several modules 
3. have used best practices
4. Good git flow

But there are some changes need to be done.
1. fix linters


see the comment section for suggestions.

File: src/index.js (hunk 1 above)
Comment (MesakDuduCoder) [reply src/index.js:41]: Thank you @yashodhicy I will make the changes.

File: src/index.js (hunk 3 above)
Comment (MesakDuduCoder) [reply src/index.js]: Thank you @yashodhicy I will make the changes.

File: src/index.js (hunk 2 above)
Comment (MesakDuduCoder) [reply src/index.js]: Thank you @yashodhicy I will make the changes.

**Review by yashodhicy — APPROVED** [review]
Well done on your work. It is time to merge.
congrats

**Pull request merged**